warehouse/templates/packaging/detail.html

```diff
@@ -71,7 +71,7 @@ <h3 class="title"><a href="{{ request.route_path('packaging.project', name=relea
 
     <p class="pip-instructions">
       <span id="pip-command">pip install {{ release.project.normalized_name }}</span>
-      <a href="#" class="copy-pip-command" data-clipboard-target="#pip-command">
+      <a href="#" class="copy-pip-command tooltipped tooltipped-s" data-clipboard-target="#pip-command" aria-label="Copy to clipboard">
```

Comment:
also, can we please namespace the js hook with `-js`. e.g `-js-copy-pip-command`.